astro

```diff
@@ -192,7 +192,7 @@ fetch() {
 
 		# Success
 		[ "$debug" ] && echo "Success, reading body"
-		[ -f "$cachedir/links.txt" ] && rm "$cachedir/links.txt"
+		[ -f "$linkfile" ] && rm "$linkfile"
```

Comment:
I think is better to call it `linksfile` instead of `linkfile` as the file is *$cachedir/links*.